src/lib/style/components/_cards.scss

```diff
@@ -31,6 +31,10 @@ $text-color-lighter: #898989;
   .SRC-portalCardMain {
     display: flex;
   }
+  .SRC-portalCardCTAButton {
```

Comment:
Ability to style this more in the future (named in the same way as other generic card elements in this scss)